docs/organize.md

```
@@ -1 +1,149 @@
+For new contributors who create their first issue, open their first
+PR, or merge their first PR, we use a welcome bot to appreciate their
+contributions. The configuration for the bot is inside
+`.github/config.yml` and can be modified if needed.
+
+The bot is not enabled by default for new projects. To enable it, you need
+to go to the [GitHub site](https://probot.github.io/apps/welcome/) and press
+the `+ Add to GitHub` button. Afterward, select your project to add it.
+
+## Automatic Labeling
+
+To automatically label pull requests, the template provides out-of-the-box configuration for a [Labeler Action](https://github.com/actions/labeler) through the workflow file in `.github/workflows/pr_labeler.yml` 
```

Markdownlint (0.37.0)

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)